spec2sdk/models/entities.py

```diff
@@ -238,25 +238,22 @@ def create_model_field_view(field: ModelField) -> ModelFieldView:
         )
 
 
-class OptionalType(PythonType):
-    inner_py_type: PythonType
-
+class NoneType(PythonType):
     @property
     def dependency_types(self) -> Sequence[PythonType]:
-        return (self.inner_py_type,)
+        return ()
 
     @property
     def type_hint(self) -> str:
-        return self.name or f"{self.inner_py_type.type_hint} | None"
+        return self.name or "None"
 
     @property
     def imports(self) -> Sequence[Import]:
         return ()
 
     def render(self) -> str:
         if self.name:
-            root_type = f"{self.inner_py_type.type_hint} | None"
-            return f"type {self.name} = {root_type}"
+            return f"type {self.name} = None"
```

Comment:
Here I had the same idea as above that seems to be a bit frowned up on:
```suggestion
            return f"type {self.name} = NoneType"
```